### ISSUE TITLE
Why not create issues like todo list for your task of xinfab?

### ISSUE THREAD

**Issue by emj365** [xinfab/Docs#4]
I just joined one of your meetings.

I saw meeting note in this repository. I suggest use issue to manage tasks of your works.

The Issue is designed to do this. So you will have a workflow can track everyone.

@evawxiao checkbox in MD file isn't good enough because it needs one person to maintain it. This person has to follow everyone's work.

The issue can let each person update their status. Everyone can see the history and track the changes.

You have free a tool but haven't used it.

**Comment by emj365** [comment]
You can check here to see how Issue works. https://github.com/xinfab/Website/issues?q=is%3Aissue+is%3Aclosed

**Comment by lpgn** [comment]
I agree but we come back to training on git because it is a bit too technical to most people.

**Comment by emj365** [comment]
Use Github is not technical. I mean this web interface but not git commadn line. We're using issues now but just type and click right?

**Comment by emj365** [comment]
**You can do something like this:**
- [x] open browser
- [ ] open GitHub
- [ ] use issues
- [ ] check notifications for updates
- [ ] get notify in email

~~it is a bit too technical to most people.~~

:smile: :tada: :rainbow:

**Comment by emj365** [comment]
Just suggestion. Because I saw https://github.com/xinfab/Docs/blob/master/Minutes/Meeting-Minutes-2016-01-05.md

**Comment by lpgn** [comment]
I am the first to vote for use Git for this. But we will have a learning curve. Even on Github website in order to write text nicely you need to do a bit o coding as in here: https://guides.github.com/features/mastering-markdown/ . So its not like writing in a word document and  YES not that hard.

So I m for it, from next staff meeting.

**Comment by emj365** [comment]
You're right. Maybe I'm too rush.
